test/integration/targets/podman_volume_info/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+- name: Test podman_volume_info
+  when:
+    - ansible_facts.virtualization_type != 'docker'
+    - ansible_facts.distribution == 'RedHat'
+  block:
```

Comment:
You want to add an `always:` part which always cleans up the volume. Otherwise (when testing locally), every failed run might leave a volume.

Reply:
thanks, will add